apps/web/src/components/transactions/TxDetails/index.tsx

```diff
@@ -175,6 +176,8 @@ const TxDetailsBlock = ({ txSummary, txDetails }: TxDetailsProps): ReactElement
             proposer={proposer}
           />
 
+          <HnSecurityReportBtnForTxDetails txDetails={txDetails} />
```

Comment:
TODO: rearrange the components here so that the TxShareBlock is in top-right corner.